src/main/java/fr/insee/pearljam/infrastructure/surveyunit/entity/CommunicationRequestDB.java

```diff
@@ -55,16 +61,32 @@ public class CommunicationRequestDB implements Serializable {
     @OneToMany(fetch = FetchType.LAZY, targetEntity = CommunicationRequestStatusDB.class, cascade = CascadeType.ALL, mappedBy = "communicationRequest", orphanRemoval = true)
     private List<CommunicationRequestStatusDB> status;
 
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "campaign_id", referencedColumnName = "campaign_id", insertable = false, updatable = false)
+    @JoinColumn(name = "meshuggah_id", referencedColumnName = "meshuggah_id", insertable = false, updatable = false)
+    private CommunicationTemplateDB communicationTemplate;
+
+    public CommunicationRequestDB(Long id, CommunicationTemplateDBId communicationTemplateDBId,
+        CommunicationRequestReason reason, CommunicationRequestEmitter emitter,
+        SurveyUnit surveyUnit, List<CommunicationRequestStatusDB> status) {
+        this.id = id;
+        this.communicationTemplateDBId = communicationTemplateDBId;
+        this.reason = reason;
+        this.emitter = emitter;
+        this.surveyUnit = surveyUnit;
+        this.status = status;
+    }
+
     /**
      * Create entity object from model object
      * @param surveyUnit survey unit entity
      * @param request model object
      * @return entity object
      */
-    public static CommunicationRequestDB fromModel(CommunicationRequest request, SurveyUnit surveyUnit, CommunicationTemplateDB communicationTemplate) {
+    public static CommunicationRequestDB fromModel(CommunicationRequest request, SurveyUnit surveyUnit, String campaignId, String meshuggahId) {
```

Comment:
```suggestion
    public static CommunicationRequestDB fromModel(CommunicationRequest request, SurveyUnit surveyUnit, CommunicationTemplateDB communicationTemplate) {
```

then make new CommunicationRequestDB with communicationTemplate.getComTemDbId()